zfit_physics/models/pdf_erfexp.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+import tensorflow as tf
+import zfit
+from zfit import z
+from zfit.util import ztyping
+from zfit.z import numpy as znp
+
+
+@z.function(wraps="tensor")
+def erfexp_pdf_func(x, alpha, beta, gamma, n):
+    """Calculate the ErfExp PDF.
+
+    Args:
+        x: value(s) for which the PDF will be calculated.
+        alpha: Location parameter.
+        beta: Scale parameter.
+        gamma: Shape parameter.
+        n: Shape parameter.
+
+    Returns:
+        `tf.Tensor`: The calculated PDF values.
+
+    Notes:
+        Implementation from https://gitlab.cern.ch/cms-muonPOG/spark_tnp/-/blob/Spark3/RooErfExp.cc
+        The parameters beta and gamma are given in reverse order in this c++ implementation.
+    """
+    return tf.math.erfc((x - alpha) * beta) * znp.exp(-gamma * (znp.power(x, n) - znp.power(alpha, n)))
```

Comment:
about names, why not use `mu` instead of alpha and _divide_ by beta, which we also rename to `sigma`? I think that's a lot more intuitive on to what the meaning of these is? Or are there any drawbacks?

Reply:
Or is that unintuitive?

Reply:
mu seemed unintuitive to me because it's not where the peak is. I'd rather keep it as is .

Reply:
That's true, good point. It's though the mu of the integrated Gaussian.
alpha and beta just seem very generic, something like `loc` and `scale` of the erf would make sense, but this is named usually `mu` and `sigma` (or mostly?). Anything that is more descriptive?
Also, dividing by the scale/beta/sigma seems more intuitive, doesn't it? The larger it gets, the wider the PDF? `beta` is currently also as "scale" parameter, but if, it's rather the "inverse" scale?
Maybe need to think about it

(btw, it's a pretty important topic to have good names: once they're in, we cannot just go and change them. We will need to deprecate, and at some point raise errors, and users may need to adjust their code. Yes, it happens. But we should just put some reasonable arguments up, ideally)